dev/com.ibm.ws.beanvalidation.v20_fat/test-applications/bvalELApp/src/bval/v20/el/web/ELBean.java

```diff
@@ -0,0 +1,26 @@
+/*******************************************************************************
+ * Copyright (c) 2024 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License 2.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-2.0/
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package bval.v20.el.web;
+
+import javax.enterprise.context.ApplicationScoped;
+
+/**
+ * CDI Bean with a single string for Expression language testing
+ */
+@ApplicationScoped
+public class ELBean {
+
+    @ELInvalid
+    String testString = "test";
```

Comment:
This is our CDI bean, with the `@ELInvalid` custom validation annotation made for this test.